src/components/mobile/mobile-next-query-preview.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <NextQueryPreview
```

Comment:
The list of NQs does not take the full width available like in the designs.

![image](https://user-images.githubusercontent.com/68222542/183625287-09dbdf04-a209-425a-9aa6-485fbda99861.png)
VS
![image](https://user-images.githubusercontent.com/68222542/183625391-23a90739-fb31-4c59-8380-b3cb1eaec4a7.png)

Reply:
I think this is because of the wrong margin/paddings

You can set a negative margin of `24px` (The 06 value of our design system sizes) to match parent containers 24px padding, and then set a padding left of 24px too in the scroll element of the sliding panel

Reply:
Which browser are you using? I see it taking the full width

<img width="357" alt="image" src="https://user-images.githubusercontent.com/6247440/183634463-643f8a8f-5bc7-4b0e-b969-55dd46f0ab0e.png">